plugins/redacted/tracker.py

```diff
@@ -152,3 +152,7 @@ def get_download_location_context(self, torrent_info):
             }
         except RedactedTorrent.DoesNotExist:
             return {}
+
+    def get_snatched(self, offset, limit):
```

Comment:
Does it have to be in the tracker? Why not call RedactedClient directly, like most other places do? You can instantiate the client and use it directly from your command and avoid adding to the tracker API.